diff --git a/node_modules/expo-av/android/build.gradle b/node_modules/expo-av/android/build.gradle
index dc15779..972338e 100644
--- a/node_modules/expo-av/android/build.gradle
+++ b/node_modules/expo-av/android/build.gradle
@@ -1,4 +1,4 @@
-import java.nio.file.Paths
+﻿import java.nio.file.Paths
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
@@ -21,7 +21,6 @@ def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.ab
 if (expoModulesCorePlugin.exists()) {
   apply from: expoModulesCorePlugin
   applyKotlinExpoModulesCorePlugin()
-  // Remove this check, but keep the contents after SDK49 support is dropped
   if (safeExtGet("expoProvidesDefaultConfig", false)) {
     useExpoPublishing()
     useCoreDependencies()
@@ -29,12 +28,9 @@ if (expoModulesCorePlugin.exists()) {
 }
 
 buildscript {
-  // Simple helper that allows the root project to override versions declared by this library.
   ext.safeExtGet = { prop, fallback ->
     rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
   }
-
-  // Ensures backward compatibility
   ext.getKotlinVersion = {
     if (ext.has("kotlinVersion")) {
       ext.kotlinVersion()
@@ -42,11 +38,9 @@ buildscript {
       ext.safeExtGet("kotlinVersion", "1.8.10")
     }
   }
-
   repositories {
     mavenCentral()
   }
-
   dependencies {
     classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${getKotlinVersion()}")
   }
@@ -69,49 +63,22 @@ if (!safeExtGet("expoProvidesDefaultConfig", false)) {
     }
   }
 }
-
+"scripts": {
+  "postinstall": "patch-package",
+  "start": "npx expo start",
+  "android": "npx expo start --android",
+  "ios": "npx expo start --ios",
+  "web": "npx expo start --web",
+  "build:web": "npx expo export:web",
+  "vercel-build": "npx expo export:web",
+  "serve": "npx serve web-build"
+}
 android {
-  // Remove this if and it's contents, when support for SDK49 is dropped
-  if (!safeExtGet("expoProvidesDefaultConfig", false)) {
-    compileSdkVersion safeExtGet("compileSdkVersion", 34)
-
-    defaultConfig {
-      minSdkVersion safeExtGet("minSdkVersion", 23)
-      targetSdkVersion safeExtGet("targetSdkVersion", 34)
-    }
-
-    publishing {
-      singleVariant("release") {
-        withSourcesJar()
-      }
-    }
-
-    lintOptions {
-      abortOnError false
-    }
-  }
+  compileSdkVersion 34
 
-  if (rootProject.hasProperty("ndkPath")) {
-    ndkPath rootProject.ext.ndkPath
-  }
-  if (rootProject.hasProperty("ndkVersion")) {
-    ndkVersion rootProject.ext.ndkVersion
-  }
-
-  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
-  if (agpVersion.tokenize('.')[0].toInteger() < 8) {
-    compileOptions {
-      sourceCompatibility JavaVersion.VERSION_11
-      targetCompatibility JavaVersion.VERSION_11
-    }
-
-    kotlinOptions {
-      jvmTarget = JavaVersion.VERSION_11.majorVersion
-    }
-  }
-
-  namespace "expo.modules.av"
   defaultConfig {
+    minSdkVersion 24
+    targetSdkVersion 34
     versionCode 24
     versionName "13.10.6"
 
@@ -136,8 +103,6 @@ android {
   }
 
   packagingOptions {
-    // Gradle will add cmake target dependencies into packaging.
-    // Theses files are intermediated linking files to build reanimated and should not be in final package.
     excludes += [
         "**/libc++_shared.so",
         "**/libreactnativejni.so",
@@ -150,7 +115,6 @@ android {
     ]
   }
 
-
   testOptions {
     unitTests.all {
       useJUnitPlatform()
@@ -159,24 +123,18 @@ android {
 }
 
 dependencies {
-  // Remove this if and it's contents, when support for SDK49 is dropped
   if (!safeExtGet("expoProvidesDefaultConfig", false)) {
     implementation project(':expo-modules-core')
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
   }
 
   implementation 'com.facebook.react:react-android'
-
   compileOnly 'com.facebook.fbjni:fbjni:0.3.0'
-
   api 'com.google.android.exoplayer:exoplayer:2.18.1'
   api 'com.google.android.exoplayer:extension-okhttp:2.18.1'
-
   api "com.squareup.okhttp3:okhttp:3.14.9"
   api "com.squareup.okhttp3:okhttp-urlconnection:3.14.9"
-
   testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.1'
   testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.5.1"
-
   testImplementation 'io.mockk:mockk:1.12.3'
 }
\ No newline at end of file
diff --git a/node_modules/expo-av/android/build.gradle.backup b/node_modules/expo-av/android/build.gradle.backup
new file mode 100644
index 0000000..86f7806
--- /dev/null
+++ b/node_modules/expo-av/android/build.gradle.backup
@@ -0,0 +1,174 @@
+import java.nio.file.Paths
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven-publish'
+
+group = 'host.exp.exponent'
+version = '13.10.6'
+
+def REACT_NATIVE_BUILD_FROM_SOURCE = findProject(":ReactAndroid") != null
+def REACT_NATIVE_DIR = REACT_NATIVE_BUILD_FROM_SOURCE
+  ? findProject(":ReactAndroid").getProjectDir().parent
+  : new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).parent
+
+def reactNativeArchitectures() {
+    def value = project.getProperties().get("reactNativeArchitectures")
+    return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
+}
+
+def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
+if (expoModulesCorePlugin.exists()) {
+  apply from: expoModulesCorePlugin
+  applyKotlinExpoModulesCorePlugin()
+  // Remove this check, but keep the contents after SDK49 support is dropped
+  if (safeExtGet("expoProvidesDefaultConfig", false)) {
+    useExpoPublishing()
+    useCoreDependencies()
+  }
+}
+
+buildscript {
+  // Simple helper that allows the root project to override versions declared by this library.
+  ext.safeExtGet = { prop, fallback ->
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+  }
+
+  // Ensures backward compatibility
+  ext.getKotlinVersion = {
+    if (ext.has("kotlinVersion")) {
+      ext.kotlinVersion()
+    } else {
+      ext.safeExtGet("kotlinVersion", "1.8.10")
+    }
+  }
+
+  repositories {
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${getKotlinVersion()}")
+  }
+}
+
+// Remove this if and it's contents, when support for SDK49 is dropped
+if (!safeExtGet("expoProvidesDefaultConfig", false)) {
+  afterEvaluate {
+    publishing {
+      publications {
+        release(MavenPublication) {
+          from components.release
+        }
+      }
+      repositories {
+        maven {
+          url = mavenLocal().url
+        }
+      }
+    }
+  }
+}
+android {
+  compileSdk 34
+
+  defaultConfig {
+    minSdkVersion 24
+    targetSdkVersion 34
+    versionCode 24
+    versionName "13.10.6"
+
+    externalNativeBuild {
+      cmake {
+        cppFlags "-O2 -frtti -fexceptions -Wall -fstack-protector-all"
+        abiFilters (*reactNativeArchitectures())
+        arguments "-DANDROID_STL=c++_shared",
+            "-DREACT_NATIVE_DIR=${REACT_NATIVE_DIR}"
+      }
+    }
+  }
+
+  externalNativeBuild {
+    cmake {
+      path "CMakeLists.txt"
+    }
+  }
+
+  buildFeatures {
+    prefab true
+  }
+
+  packagingOptions {
+    excludes += [
+        "**/libc++_shared.so",
+        "**/libreactnativejni.so",
+        "**/libglog.so",
+        "**/libjscexecutor.so",
+        "**/libfbjni.so",
+        "**/libfolly_json.so",
+        "**/libhermes.so",
+        "**/libjsi.so",
+    ]
+  }
+
+  testOptions {
+    unitTests.all {
+      useJUnitPlatform()
+    }
+  }
+}
+  externalNativeBuild {
+    cmake {
+      path "CMakeLists.txt"
+    }
+  }
+
+  buildFeatures {
+    prefab true
+  }
+
+  packagingOptions {
+    // Gradle will add cmake target dependencies into packaging.
+    // Theses files are intermediated linking files to build reanimated and should not be in final package.
+    excludes += [
+        "**/libc++_shared.so",
+        "**/libreactnativejni.so",
+        "**/libglog.so",
+        "**/libjscexecutor.so",
+        "**/libfbjni.so",
+        "**/libfolly_json.so",
+        "**/libhermes.so",
+        "**/libjsi.so",
+    ]
+  }
+
+
+  testOptions {
+    unitTests.all {
+      useJUnitPlatform()
+    }
+  }
+}
+
+dependencies {
+  // Remove this if and it's contents, when support for SDK49 is dropped
+  if (!safeExtGet("expoProvidesDefaultConfig", false)) {
+    implementation project(':expo-modules-core')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
+  }
+
+  implementation 'com.facebook.react:react-android'
+
+  compileOnly 'com.facebook.fbjni:fbjni:0.3.0'
+
+  api 'com.google.android.exoplayer:exoplayer:2.18.1'
+  api 'com.google.android.exoplayer:extension-okhttp:2.18.1'
+
+  api "com.squareup.okhttp3:okhttp:3.14.9"
+  api "com.squareup.okhttp3:okhttp-urlconnection:3.14.9"
+
+  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.1'
+  testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.5.1"
+
+  testImplementation 'io.mockk:mockk:1.12.3'
+}
